modules/subdyn/src/SubDyn.f90

```diff
@@ -1152,13 +1152,45 @@ SUBROUTINE SD_Input(SDInputFile, Init, p, ErrStat,ErrMsg)
 if (LegacyFormat) then
    nColumns = 5
    Init%Members(:,iMType) = idMemberBeam ! Important, in legacy all members are beams
+   Init%Members(:,7) = -1
 else
    nColumns = MembersCol
 endif
-DO I = 1, p%NMembers
+
```

Comment:
We can add a fatal statement here if the number of member is zero. (which indeed woulc be an invalid SubDyn model). 
Then we can be sure that we read the "first line" of the Member table below, because it should always exists.

Reply:
Agreed